WordPress/src/main/AndroidManifest.xml

```diff
@@ -113,7 +113,18 @@
         <activity
             android:name=".ui.accounts.LoginActivity"
             android:theme="@style/LoginTheme.TransparentSystemBars"
-            android:windowSoftInputMode="adjustResize" />
+            android:windowSoftInputMode="adjustResize"
+            android:exported="true">
```

Comment:
Sonarcloud flags this as a security risk. Does it need to be [exported](https://developer.android.com/privacy-and-security/risks/android-exported)?

Reply:
My understanding is that this was required in order to handle custom URL schemes, but I might be incorrect!

Reply:
Ah, I suspect you're right about that. [This Sonarcloud page](https://sonarcloud.io/organizations/wordpress-mobile/rules?open=xml%3AS5594&rule_key=xml%3AS5594&tab=how_to_fix) mentions ways to fix this warning.